components/organisation-unit-tree/src/organisation-unit-node/use-org-data/use-org-data.js

```diff
@@ -0,0 +1,70 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import { useState } from 'react'
+import { sortNodeChildrenAlphabetically } from '../../helpers/index.js'
+
+const ORG_DATA_QUERY = {
+    orgUnit: {
+        resource: `organisationUnits`,
+        id: ({ id }) => id,
+        params: ({ isUserDataViewFallback }) => ({
+            isUserDataViewFallback,
+            fields: ['id', 'path', 'children[id,path,displayName]'],
+            paging: false,
```

Comment:
Since this is a query for a singular org unit is `paging: false` needed?

Reply:
You're right, can be omitted. It's a leftover from when this hook was used for two different use cases